dependencies/requirements-dev.in

```diff
@@ -0,0 +1,9 @@
+black==22.3.0
+flake8==4.0.1
+flake8-bugbear==22.4.25
+isort==5.10.1
+mypy==0.950
+mypy-extensions==0.4.3
+pip-tools==6.6.1
+types-requests==2.27.27
+types-python-dateutil==2.8.16
```

Comment:
do we need this module in here?

Reply:
I added these, because `mypy` reported:
```
src\price_data.py:26:1: error: Library stubs not installed for "requests" (or incompatible with Python 3.9)  [import]
src\price_data.py:26:1: note: Hint: "python3 -m pip install types-requests"
src\price_data.py:26:1: note: (or run "mypy --install-types" to install all missing stub packages)
src\price_data.py:26:1: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
src\taxman.py:25:1: error: Library stubs not installed for "dateutil.relativedelta" (or incompatible with Python 3.9)  [import]
src\taxman.py:25:1: note: Hint: "python3 -m pip install types-python-dateutil"
```

`mypy` can probably be configured to not check types on these modules.